src/main/java/net/openhft/chronicle/core/Maths.java

```diff
@@ -285,17 +286,31 @@ public static long hash64(@NotNull CharSequence cs) {
 
     public static long hash64(@NotNull String s) {
```

Comment:
Please add a unit test to MathsTest that exercises these code paths.

Reply:
@epickrram https://github.com/OpenHFT/Chronicle-Core/pull/37/files#diff-b2162a42c901ba85c335d409e8a97350R323 - Does the test look alright to you?

Reply:
Looks good.